packages/store/docs/exo-taxonomy.md

```diff
@@ -1,46 +1,55 @@
 
-- `makeExo`
-- `defineExoClass`
-- `defineExoClassKit`
+### VirtualExo*
+Like the big stores, the virtual exo objects created by the functions returned by `defineVirtualExoClass` or `defineVirtualExoClassKit` are written to external storage outside the JavaScript heap. But these are ephemeral -- they do not survive upgrade. Their only purpose is for high cardinality, so we do not provide a convenience for directly making exo instances. IOW, there is no `makeVirtualExo`.
+
+### DurableExo*
+The durable exo objects created by the functions returned by `defineDurableExoClass` or `defineDurableExoClassKit` are also written to external storage. These can also survive upgrade, and so can be passed in baggage to a successor vat-incarnation. Note that there is no `makeDurableExo`, although the need is largely covered by `prepareExo`.
 
-The virtual and durable variants are contributed by higher layer packages that build on this one, such as `@agoric/vat-data`.
+### Class Cardinality
+The total number of exo classes must be low cardinality, regardless of virtual/durable status. Being virtual or durable only enables class *instances* to be high cardinality.
 
-## Make/Define vs Prepare (Durable only)
+## Make/Define vs Prepare
 
 "prepare" is like "provide" in that it defines something that should be in the baggage, using the one that is there if found, but otherwise making a new one and registering it, so that the successor vat-invocation will find it at the same place in the baggage. Unlike "provide", for each exo behavior already in the baggage, one must call "prepare" immediately --- during the first crank of the vat incarnation. What is passed in baggage is only the state of the durable objects. Only the `prepare*` calls associate that state with code, giving it behavior. All these objects must be prepared early, so they know how to react when they receive messages.
 
-- **_`prepareExo`_** <br>
-  Like `makeExo` but for a durable exo in baggage.
-- **_`prepareExoClass`_** <br>
-  Like `defineExoClass` but for a durable exo in baggage.
-- **_`prepareExoClassKit`_** <br>
-  Like `defineExoClassKit` but for a durable exo in baggage.
+- `prepareExo`:
+  Each call returns a durable exo in baggage, creating it first if necessary.
+- `prepareExoClass`:
+  Each call returns a "make" function for the durable exo class in baggage, creating it first if necessary.
+- `prepareExoClassKit`:
+  Each call returns a "makeKit" function for the durable exo class kit in baggage, creating it first if necessary.
+<!-- end copied data -->
+
```

Comment:
This whole section is excellent, and I learned a lot reading it. It might be even better with a table, showing the function names used for each pairing of heap/virtual/durable and exp/class/kit (and showing the intentionally omitted combinations).

Reply:
I thought so too in https://github.com/endojs/endo/pull/1469 , and consider it another improvement to make upstream.